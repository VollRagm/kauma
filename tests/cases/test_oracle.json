{
    "testcases": {
        "b856d760-023d-4b00-bad2-15d2b6da22fe": {
            "action": "padding_oracle",
            "arguments": {
                "hostname": "server_simulator",
                "port": 42069,
                "iv": "AAAAAAAAAAAAAAAAAAAAAA==",
                "ciphertext": "SWdvaGomUGd7Zm8tLVpneTtFBR9PQXcbFx8QAAA7HgxIZzU7KCsfcHVmOixFWmF5aQxCH1pCahMPTTcmTlJobw=="
            }
        },
        "254eaee7-05fd-with-double-match": {
            "action": "padding_oracle",
            "arguments": {
                "hostname": "server_simulator",
                "port": 42069,
                "iv": "AAAAAAAAAAAAAAAAAAAAAA==",
                "ciphertext": "AgEABwYFBAsKCQgPDwwNEQ=="
            }
        }
    }
}