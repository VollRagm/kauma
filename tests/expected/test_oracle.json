{
    "responses": {
        "b856d760-023d-4b00-bad2-15d2b6da22fe": {
            "plaintext": "SGVsbG8gV29ybGQhIFRoaXMgaXMgYSB0ZXN0ISBvdmVyIDMgYmxvY2tzISBIb3BlIGl0IHdvcmtzIQYGBgYGBg=="
        },
        "254eaee7-05fd-with-double-match": {
            "plaintext": "AwMDAwMDAwMDAwMDAgICAQ=="
        }
    }
}